x-pack/test/case_api_integration/security_and_spaces/tests/common/cases/status/get_status.ts

```diff
@@ -24,41 +33,139 @@ export default ({ getService }: FtrProviderContext): void => {
 
   describe('get_status', () => {
     afterEach(async () => {
-      await deleteCasesByESQuery(es);
+      await deleteAllCaseItems(es);
     });
 
     it('should return case statuses', async () => {
-      await createCase(supertest, postCaseReq);
-      const inProgressCase = await createCase(supertest, postCaseReq);
-      const postedCase = await createCase(supertest, postCaseReq);
-
-      await updateCase(supertest, {
-        cases: [
-          {
-            id: postedCase.id,
-            version: postedCase.version,
-            status: CaseStatuses.closed,
-          },
-        ],
-      });
+      const [, inProgressCase, postedCase] = await Promise.all([
+        createCase(supertest, postCaseReq),
+        createCase(supertest, postCaseReq),
+        createCase(supertest, postCaseReq),
+      ]);
 
-      await updateCase(supertest, {
-        cases: [
-          {
-            id: inProgressCase.id,
-            version: inProgressCase.version,
-            status: CaseStatuses['in-progress'],
-          },
-        ],
+      await updateCase({
+        supertest,
+        params: {
+          cases: [
+            {
+              id: inProgressCase.id,
+              version: inProgressCase.version,
+              status: CaseStatuses['in-progress'],
+            },
+            {
+              id: postedCase.id,
+              version: postedCase.version,
+              status: CaseStatuses.closed,
+            },
+          ],
+        },
       });
 
-      const statuses = await getAllCasesStatuses(supertest);
+      const statuses = await getAllCasesStatuses({ supertest });
 
       expect(statuses).to.eql({
         count_open_cases: 1,
         count_closed_cases: 1,
         count_in_progress_cases: 1,
       });
     });
+
+    describe('rbac', () => {
+      const supertestWithoutAuth = getService('supertestWithoutAuth');
+
+      it('should return the correct status stats', async () => {
+        /**
+         * Owner: Sec
+         *  open: 0, in-prog: 1, closed: 1
+         * Owner: Obs
+         *  open: 1, in-prog: 1
+         */
+        const [inProgressSec, closedSec, , inProgressObs] = await Promise.all([
+          createCase(supertestWithoutAuth, getPostCaseRequest(), 200, {
+            user: superUser,
+            space: 'space1',
+          }),
+          createCase(supertestWithoutAuth, getPostCaseRequest(), 200, {
+            user: superUser,
+            space: 'space1',
+          }),
+          createCase(
+            supertestWithoutAuth,
+            getPostCaseRequest({ owner: 'observabilityFixture' }),
+            200,
+            { user: superUser, space: 'space1' }
```

Comment:
we could throw this object in a const and re-use it for all the requests?

Reply:
yeah good point 👍